### ISSUE TITLE
Documentation mismatch - building custom profiles

### ISSUE THREAD

**Issue by zithro** [netblue30/firejail#5049]
Hello, it's not a software bug I'm reporting, rather a documentation one (so I didn't use the template).
I followed "documentation-2/building-custom-profiles/" to build a vlc custom profile, as this:
```
net none
include /etc/firejail/vlc.profile
```
But when running `firejail vlc`, I enter in an include loop : 
```
Reading profile /etc/firejail/vlc.profile
Reading profile /home/zithro/.config/firejail/vlc.local
[...]
Reading profile /etc/firejail/vlc.profile
Error: maximum profile include level was reached
```
Removing the include solves it.
Is it because I'm running the latest version (0.9.68) and the documentation was written using an older version ?

**Comment by rusty-snake** [comment]
`include /etc/firejail/vlc.profile` is not needed (at wrong) if you use a .local.

**Comment by kmk3** [comment]
@zithro commented [on Mar 14](https://github.com/netblue30/firejail/issues/5049#issue-1168945593):

> Hello, it's not a software bug I'm reporting, rather a documentation one (so
> I didn't use the template). I followed
> "documentation-2/building-custom-profiles/" to build a vlc custom profile, as
> this:
> 
> ```
> net none
> include /etc/firejail/vlc.profile
> ```
> 
> But when running `firejail vlc`, I enter in an include loop :
> 
> ```
> Reading profile /etc/firejail/vlc.profile
> Reading profile /home/zithro/.config/firejail/vlc.local
> [...]
> Reading profile /etc/firejail/vlc.profile
> Error: maximum profile include level was reached
> ```
> 
> Removing the include solves it. Is it because I'm running the latest version
> (0.9.68) and the documentation was written using an older version ?

/etc/firejail/vlc.profile contains:

```
include vlc.local
```

On every include command, firejail will first look for the file in
~/.config/firejail.  If it does not exist, it will look for the file in
/etc/firejail.

If both vlc.profile and vlc.local include each other, then you have an infinite
inclusion loop.

**Comment by zithro** [comment]
Hey guys, thanks for the clarification, so I think the docs need an update ;)
"documentation-2/building-custom-profiles/" meant in fact "https://firejail.wordpress.com/documentation-2/building-custom-profiles/"
Exerpt : 
```
One popular customization is removing network access for media players such as VLC:
$ cat ~/.config/firejail/vlc.profile
net none
include /etc/firejail/vlc.profile 
```

**Comment by glitsj16** [comment]
> [...] so I think the docs need an update

Indeed they do. All the examples in that section are inclusion loops. Not sure who actually maintains https://firejail.wordpress.com/ to correct them.

**Comment by zithro** [comment]
@netblue30 is the author, look at the website comments.

**Comment by rusty-snake** [comment]
> All the examples in that section are inclusion loops.

No, because the suggest to create `vlc.profile` not `vlc.local`.
FTR: Nowadays you should create a `vlc.local` without `include /etc/firejal/vlc.profile`.

**Comment by netblue30** [comment]
I wasn't aware of the .local functionality. We just need to document the difference between .local and .profile for files in ~/.config/firejail directory.

**Comment by rusty-snake** [comment]
> Should I close this issue ?

No
